Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/adapters/GameRowPresenter.kt

```diff
@@ -7,20 +7,29 @@ import android.view.ViewGroup
 import androidx.leanback.widget.ImageCardView
 import org.dolphinemu.dolphinemu.viewholders.TvGameViewHolder
 import org.dolphinemu.dolphinemu.model.GameFile
-import org.dolphinemu.dolphinemu.utils.GlideUtils
 import org.dolphinemu.dolphinemu.services.GameFileCacheManager
 import org.dolphinemu.dolphinemu.R
 import android.view.View
 import androidx.core.content.ContextCompat
 import android.widget.ImageView
 import androidx.fragment.app.FragmentActivity
+import androidx.lifecycle.lifecycleScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
 import org.dolphinemu.dolphinemu.dialogs.GamePropertiesDialog
+import org.dolphinemu.dolphinemu.utils.CoilUtils
 
 /**
  * The Leanback library / docs call this a Presenter, but it works very
  * similarly to a RecyclerView.Adapter.
  */
-class GameRowPresenter : Presenter() {
+class GameRowPresenter(activity: FragmentActivity) : Presenter() {
+    private val mActivity: FragmentActivity
+
+    init {
+        mActivity = activity
```

Comment:
I'm not all that familiar with Kotlin yet, but is this necessary? Can `activity` not be accessed outside of the `init` block?

Reply:
I looked this up in more detail. Seems like you can't use them just anywhere, but you should be able to use it when initializing `mActivity`, like this: `private val mActivity = activity`

Or, you could change the `activity` declaration to `val activity` to make it accessible throughout the class. (Though, should it be `private val activity` then...?)

Reply:
I'll just go with `private val mActivity: FragmentActivity = activity` where applicable.